src/styles/calendar.scss

```diff
@@ -0,0 +1,46 @@
+.calendar {
+  gap: $gap;
+  padding-top: 50px;
+  display: flex;
+  flex-wrap: wrap;
+  width: $calendar-width;
+
+  &__day {
+    @include square($day-size);
+    display: flex;
+    flex-wrap: wrap;
+    box-sizing: content-box;
+    background-color: #eee;
+    border: 1px black solid;
+    transition-duration: 0.5s;
+
+    &:hover {
+      transform: translateY(-20px);
+      cursor: pointer;
+      background-color: pink;
+    }
+  }
+
+  @for $date from 1 through 31 {
+    &__day:nth-child(#{$date})::before {
+      content: "#{$date}";
+      margin: auto;
```

Comment:
what is the purpose of margin auto here?

Reply:
To have all of the date numbers centred in the middle of the calendar cells.